pygmt/figure.py

```diff
@@ -297,6 +304,14 @@ def savefig(
             :meth:`pygmt.Figure.psconvert`. Ignored if creating vector images.
         show: bool
             If ``True``, will open the figure in an external viewer.
+        worldfile : bool
+            If ``True``, will create a companion
+            `world file <https://en.wikipedia.org/wiki/World_file>`__ for the
+            figure. The world file will have the same name as the figure file
+            but with different extension (e.g. tfw for tif). See
+            https://en.wikipedia.org/wiki/World_file#Filename_extension
+            for the convention of world file extensions. This parameter only
+            works for raster image formats.
```

Comment:
(Geo)TIFF is a raster file format, but a worldfile won't be produced. Also, maybe best to just list out the supported extensions.

```suggestion
            works for non-georeferenced raster image formats (e.g. '.bmp',
            '.jpg', '.png', '.ppm', '.tif')
```

Reply:
In PR https://github.com/GenericMappingTools/pygmt/pull/2771 (commit 0c8184710ea93a59ce6e3b2027114edb53e63421), we have revised the function description to separate the list of raster and vector formats (preview: https://www.pygmt.org/dev/api/generated/pygmt.Figure.savefig.html#pygmt.Figure.savefig).

So I prefer to not list the formats again and prefer to revise the docstring to 
```
works for raster image formats (except GeoTIFF).
```

Reply:
Oh right, totally missed that. Your shorter suggestion sounds good.